docs/reference/settings.rst

```diff
@@ -239,7 +239,7 @@ See :ref:`retry`.
 ZYTE_API_SKIP_HEADERS
 =====================
 
-Default: ``["User-Agent"]``
+Default: ``["Cookie"]``
```

Comment:
`Cookie` should have been here, i.e. the documentation was out of date, the actual change was the removal of `User-Agent`, to be handled based on whether its value is used-defined or not.